styx-api-service/src/main/java/com/spotify/styx/StyxApi.java

```diff
@@ -202,8 +200,18 @@ public void create(Environment environment) {
     final WorkflowActionAuthorizer workflowActionAuthorizer =
         new WorkflowActionAuthorizer(storage, serviceAccountUsageAuthorizer, actionAuthorizer);
 
+    final Duration defaultTtl = Duration.parse(config.getString(STYX_RUNNING_STATE_DEFAULT_TTL_KEY));
```

Comment:
How about we do the same as scheduler to load the config using `TimeoutConfig` so we don't need to deal with the default value?

Reply:
sure, need to move the timeout config to styx common 👍